Dockerfile

```diff
@@ -11,9 +11,10 @@ RUN apk add --no-cache $PACKAGES
 RUN CGO_ENABLED=0 make install
 
 # Add to a distroless container
-FROM gcr.io/distroless/cc:$IMG_TAG
+FROM distroless.dev/static:$IMG_TAG
 ARG IMG_TAG
 COPY --from=gaiad-builder /go/bin/gaiad /usr/local/bin/
 EXPOSE 26656 26657 1317 9090
+USER 0
```

Comment:
Given that you've applied the patch @okwme, please remove this line as we want the code to run as non-root. After that this PR will be good to go and not have a high security surface :-) Thank you @okwme for jumping onto this ASAP.

Reply:
actually the change was made in `e2e.Dockerfile`. I added the `USER 0` to `Dockerfile` because I couldn't find any use of `Dockerfile` in our code and am not sure why it was added originally or who is using it. Until I'm able to confirm that I'd like to preserve the behavior as much as possible and previously that docker image was run from root so i'd like it to stay that way for now. 

`e2e.Dockerfile` is the docker file we're using in our e2e and it now has the relevant changes.

Reply:
Gotcha, cool!